src/core/mod.rs

```diff
@@ -257,23 +271,22 @@ impl BPF {
         BPFBuilder::new(code)?.build()
     }
 
-    // get access to the interal pointer for the bpf module
+    // get access to the internal pointer for the bpf module
     fn ptr(&self) -> *mut c_void {
         self.p.load(Ordering::SeqCst)
     }
 
     /// Get access to a named table within the running BPF program.
-    pub fn table(&self, name: &str) -> Table {
-        // TODO: clean up this unwrap (and all the rest in this file)
-        let cname = CString::new(name).unwrap();
+    pub fn table(&self, name: &str) -> Result<Table, std::ffi::NulError> {
```

Comment:
Should we return a `BccError` here instead? I'm thinking it might help avoid a breaking change in the future if we always use `BccError` type in our returns.